config/application.rb

```diff
@@ -71,15 +71,20 @@ class Application < Rails::Application
     # https://github.com/mastodon/mastodon/pull/24241#discussion_r1162890242
     config.active_support.cache_format_version = 6.1
 
-    config.add_autoload_paths_to_load_path = false
-
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration should go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded.
-
-    # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
-    # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    # config.time_zone = 'Central Time (US & Canada)'
+    # Please, add to the `ignore` list any other `lib` subdirectories that do
+    # not contain `.rb` files, or that should not be reloaded or eager loaded.
+    # Common ones are `templates`, `generators`, or `middleware`, for example.
+    # config.autoload_lib(ignore: %w(assets tasks templates generators))
```

Comment:
This is actually a cool feature - https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#config-autoload-lib-ignore - but we have a pretty big collection of files that are not where zeitwerk expects them ... some in lib should be in config/initializers I think. Others just need to be moved. Makes sense to me to leave this turned off for now, work through the file renames separately, then enable this when ready.